lib/rubocop/ast/node.rb

```diff
@@ -464,7 +464,10 @@ def chained?
       end
 
       def argument?
-        parent && parent.send_type? && parent.arguments.include?(self)
+        # TODO: Adds support safe navigator calls.
+        # TODO: Discuss with maintainers if this should be applied 'everywhere'
+        parent && (parent.send_type? || parent.csend_type?) &&
```

Comment:
This has greater implications for other cops, and I'd like some input from maintainers here.

I _think_ that `Node#argument?` also accepting `:csend` is more correct than just `:send`, but I'm not too well versed in this project to know all possible implications of this.

If this is the correct course, `Node#chained?` would also change in similar fashion. (and I'll have to add test for this of course.)